conbench/types.py

```diff
@@ -0,0 +1,11 @@
+# Do this for case ID, etc, too.
+
+from typing import NewType, Tuple
+
+TBenchmarkName = NewType("TBenchmarkName", str)
```

Comment:
`T` = "type"?

Reply:
Yes. This might be the start of introducing a convention (already did that before around job.py).